### PR TITLE
Channel edits continue: Syncing tags, matching logos

### PR DESCRIPTION
**Pull request by jkemp814**
Sanitation continues

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -6410,17 +6410,17 @@ https://lnc-witi-fox-aws.tubi.video/index.m3u8
...
 https://content.uplynk.com/40cec2bf074c40f08932da03ab4510be.m3u8
 #EXTINF:-1 tvg-id="WLNGRadio.us" tvg-name="WLNG Radio" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/11r6tgW.png" group-title="Local",WLNG Radio
 http://wlngstudiowebcam.srfms.com:1935/wlngstudiowebcam/livestream/playlist.m3u8
-#EXTINF:-1 tvg-id="WMFD.us" tvg-name="WMFD" tvg-country="US" tvg-language="English" tvg-logo="https://wmfd.com/images/logo.png" group-title="Local",WMFD (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="WMFDTV.us" tvg-name="WMFD-TV 68 Mansfield" tvg-country="US" tvg-language="English" tvg-logo="https://wmfd.com/images/logo.png" group-title="Local",WMFD-TV 68 Mansfield (720p) [Not 24/7]
 https://player-api.new.livestream.com/accounts/1194911/events/3466400/live.m3u8
-#EXTINF:-1 tvg-id="WMUR.us" tvg-name="WMUR New Hampshire" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/6oNx5F0.png" group-title="Local",WMUR New Hampshire
+#EXTINF:-1 tvg-id="WMUR.us" tvg-name="ABC 9 New Hampshire (WMUR-TV)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/6oNx5F0.png" group-title="Local",ABC 9 New Hampshire (WMUR-TV)
```
Comment (LaneSh4d0w) [inline channels/us.m3u:6419]: I suppose you wanted to put WMURTV.us on tvg-id

```suggestion
#EXTINF:-1 tvg-id="WMURTV.us" tvg-name="ABC 9 New Hampshire (WMUR-TV)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/6oNx5F0.png" group-title="Local",ABC 9 New Hampshire (WMUR-TV)
```

Comment (jkemp814) [reply channels/us.m3u:6419]: Good catch

File: channels/us.m3u [hunk 2]
```diff
@@ -2124,7 +2124,7 @@ https://lnc-ktbc-fox-aws.tubi.video/index.m3u8
 https://content.uplynk.com/channel/6cbf2d32a5384dc1b787539b1102433c.m3u8
 #EXTINF:-1 tvg-id="KTLA5.us" tvg-name="KTLA 5 (Los Angeles)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/JH99Psi.jpg" group-title="Local",KTLA 5 (Los Angeles) (720p)
 https://bozztv.com/teleyupp1/teleup-N8qwnqgUq2/playlist.m3u8
-#EXTINF:-1 tvg-id="KTVKDT5.us" tvg-name="KTVK-DT5" tvg-country="US" tvg-language="English" tvg-logo="https://meredith-img.s3.amazonaws.com/listings/KTVK-DT5.png" group-title="",KTVK-DT5 (480i)
+#EXTINF:-1 tvg-id="KTVKDT5.us" tvg-name="ABC 3.5 Arizona Weather Authority (KTVK-DT5)" tvg-country="US" tvg-language="English" tvg-logo="https://meredith-img.s3.amazonaws.com/listings/KTVK-DT5.png" group-title="",ABC 3.5 Arizona Weather Authority (KTVK-DT5) (480i)
```
Comment (Dum4G) [inline channels/us.m3u:2127]: Weather category

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 2 above)
Comment (jkemp814) [reply channels/us.m3u:2127]: Another good catch

**Review by Dum4G — APPROVED** [review]

**Pull request merged**